### DISCUSSION TITLE
Silly issue with HTML page loading / redirection

### DISCUSSION THREAD

**Discussion by Yoshimano** [codesandbox/codesandbox-client#4757] [General]
Hi guys, 

I'm new to Codesandbox and am probably missing the point totally.

I just created a project for kicks (training purposes) and have a couple of HTML pages in my project, one of which being "index.html".
These pages share a top navigation section with hyperlinks (anchor elements), and whenever I try to display any other page than "index.html", the page briefly loads (something like half a second) then displays the content of index.html, while the page URL remains unchanged (the URL of the actual page I'm trying to access).

This seems like behavior specific to CodeSandBox, as I have tried loading these pages in other browsers (Firefox, Chrome) without encountering this.

Any idea what could cause this: my HTML code, CodeSandBox browser, stupidity... ?

Cheers!

**Comment by christianalfoni** [comment]
Hi there!

You can track an issue on it here: https://github.com/codesandbox/codesandbox-client/issues/4857 😄 

I will let you know when it is fixed though!

**Reply by Yoshimano** [reply]
Hi, thanks for letting me know!

Will do


Cheeers,
Sébastien

**Comment by christianalfoni** [comment]
I believe this has been fixed now, but please let us know if there are any issues still 😄
